_data/projects/.net-compiler-platform-roslyn.yml

```diff
@@ -0,0 +1,15 @@
+---
+name: .NET Compiler Platform ("Roslyn")
+desc: The .NET Compiler Platform ("Roslyn") provides open-source C# and Visual Basic
+  compilers with rich code analysis APIs.
+site: https://github.com/dotnet/roslyn
+tags:
+- roslyn
+- csx
+- script
+- compiler
+- C#
+- VB.NET
```

Comment:
Not sure how much control I have over the "export to YAML" step. I'll look into this.